lib/buffer.js

```diff
@@ -295,6 +295,17 @@ function byteLength(string, encoding) {
 Buffer.byteLength = byteLength;
 
 
+Object.defineProperty(Buffer.prototype, 'dataView', {
+  get: function() {
```

Comment:
Right, my bad. So the body should call `Object.defineProperty(this, 'dataView', { value: ... });`
I don't like the fact that we need two properties (dataView and _dataView) for one.